### PR TITLE
Added Top Line Pie Chart and Metric Tiles to Team Page

### PR DESCRIPTION
**Pull request by barbosab**
Screen Shot 2022-05-10 at 1 14 34 PM

### CODE REVIEW

File: src/UserPullPieChart.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        pullAuthorCounter[pull.author].value + 1;
+      pullAuthorCounter[pull.author].totalCodeChanges =
+        pullAuthorCounter[pull.author].totalCodeChanges + pull.totalCodeChanges;
+    } else {
+      pullAuthorCounter[pull.author] = {
+        value: 1,
+        totalCodeChanges: pull.totalCodeChanges,
+        color: COLORS[colorCounter],
+      };
+      colorCounter++;
```
Comment (blamattina) [inline src/UserPullPieChart.tsx:42]: What happens if there are more than 9 people on a team?

File: src/UserPullPieChart.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  authoredPullRequests: PullRequestKeyMetrics[];
+};
+
+function getPullPieData(pullRequests: PullRequestKeyMetrics[]) {
+  const pullAuthorCounter: any = {};
+  let colorCounter = 0;
+  pullRequests.forEach((pull) => {
+    if (pullAuthorCounter[pull.author]) {
+      pullAuthorCounter[pull.author].value =
+        pullAuthorCounter[pull.author].value + 1;
```
Comment (blamattina) [inline src/UserPullPieChart.tsx:33]: nit:
```suggestion
      pullAuthorCounter[pull.author].value++;
```

File: src/UserPullPieChart.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      pullAuthorCounter[pull.author].totalCodeChanges =
+        pullAuthorCounter[pull.author].totalCodeChanges + pull.totalCodeChanges;
```
Comment (blamattina) [inline src/UserPullPieChart.tsx:35]: nit:
```suggestion
      pullAuthorCounter[pull.author].totalCodeChanges += pull.totalCodeChanges;
```

File: src/UserPullPieChart.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        totalCodeChanges: pull.totalCodeChanges,
+        color: COLORS[colorCounter],
+      };
+      colorCounter++;
+    }
+  });
+
+  const authorPieData = Object.keys(pullAuthorCounter).map((key) => {
+    return {
+      name: key,
```
Comment (blamattina) [inline src/UserPullPieChart.tsx:48]: If we assign `name: key` on L37 we could we get away with `Object.entries(pullAuthorCounter)` here?

**Review by blamattina — APPROVED** [review]
A few suggestions but otherwise LGTM

File: src/UserPullPieChart.tsx (hunk 1 above)
Comment (barbosab) [reply src/UserPullPieChart.tsx:42]: Interestingly it doesn't crash...

Screen Shot 2022-05-11 at 10 13 35 AM

Comment (barbosab) [reply src/UserPullPieChart.tsx:40]: Added a check so it will now start reusing the colors.

**Pull request merged**